runtime/bcverify/rtverify.c

```diff
@@ -401,17 +400,187 @@ matchStack(J9BytecodeVerificationData * verifyData, J9BranchTargetStack *liveSta
 	goto _finished;
 }
 
-
-/* 
-	Walk the bytecodes linearly and verify that the recorded stack maps match.
-
-	returns BCV_SUCCESS on success
-	returns BCV_ERR_INTERNAL_ERROR on verification error
-	returns BCV_ERR_INSUFFICIENT_MEMORY on OOM
+/**
+* Return an extended NPE message.
+*
+* Note: the caller is responsible for freeing the returning string if it is not NULL.
+*
+* @param vmThread The current J9VMThread
+* @param bcCurrentPtr The pointer to the bytecode being executed and caused the NPE
+* @param romClass The romClass of the bytecode
+* @param constantPool The J9RomClass constant pool for the romClass
+* @param npeCauseMsg The extended NPE message part 2 (to be provided by another PR)
```

Comment:
The "part 2" is confusing, and please don't mention "another PR".

Reply:
fixed.